v2/bootstrap/container/common.go

```diff
@@ -0,0 +1,19 @@
+//
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package container
+
+import (
+	"github.com/edgexfoundry/go-mod-bootstrap/v2/di"
+	v2Clients "github.com/edgexfoundry/go-mod-core-contracts/v2/v2/clients/interfaces"
+)
+
+// CommonClientName contains the name of the CommonClient instance in the DIC.
+var CommonClientName = "V2CommonClient"
+
+// CommonClientFrom helper function queries the DIC and returns the CommonClient instance.
+func CommonClientFrom(get di.Get) v2Clients.CommonClient {
+	return get(CommonClientName).(v2Clients.CommonClient)
```

Comment:
Please use the pattern here so `nil` is properly retuned if DIC doesn't contain object.
https://github.com/edgexfoundry/go-mod-bootstrap/blob/master/bootstrap/container/configuration.go#L27
Same for all this implementations.

Reply:
Thanks!  Modified through https://github.com/edgexfoundry/go-mod-bootstrap/pull/158/commits/d642159ca0d71c54bb031435381dd86a003c3403